tables/vpc_flow_log/vpc_flow_log_extractor.go

```diff
@@ -32,26 +40,21 @@ func (c *VPCFlowLogExtractor) Extract(_ context.Context, a any) ([]any, error) {
 		return nil, fmt.Errorf("expected byte[], got %T", a)
 	}
 
-	mappedData, err := ConvertToMapSlice(string(jsonBytes))
+	mappedData, err := ConvertToMapSlice(c, string(jsonBytes))
 	if err != nil {
 		return nil, fmt.Errorf("error in mapping the results in the form of []map[string]string")
 	}
 
 	var res = make([]any, len(mappedData))
 	for i, record := range mappedData {
```

Comment:
Is this iteration required, previously the mappedData was iterated as a mapping was done and errors may have been reported out, I think we can just return mappedData here?

Reply:
As we are not using any mapper here, the mapped data should be of type (`*types.DynamicRow`) I have made the changes to return the `[]*types.DynamicRow{}`. Otherwisewe will get the error like `required *types.DynamicRow got map[string]string`